pallets/providers/Cargo.toml

```diff
@@ -19,6 +19,8 @@ codec = { workspace = true }
 scale-info = { workspace = true }
 
 # Local
+pallet-proofs-dealer = { workspace = true }
```

Comment:
You're tightly coupling it. Put it in `dev-dependencies`

Reply:
thanks for spotting